datastores-us-central1/k8s/templates/vars/main.yaml

```diff
@@ -0,0 +1,5 @@
+---
+namespace: anjali-training
+_public_key: c06c703ce133865d57ccbe38857b53ac44ff086f0f8f02caff29730cc1bb7414
+secrets:
+  mariadb-secret:
```

Comment:
you need to include the full secret here. it will fail to template if there is no value. normally we would not hardcode this in the vars file, but for now it is okay.